worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java

```diff
@@ -480,6 +501,27 @@ default IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set, boolean
         }
     }
 
+    private boolean isProcessExtra(IChunkSet set, boolean processExtra, int layer, Object arr) {
+        for (int y = 0, index = 0; y < 16; y++) {
+            for (int z = 0; z < 16; z++) {
+                for (int x = 0; x < 16; x++, index++) {
+                    if (!Array.get(arr, index).equals(0) && contains(x, y, z)) {
+                        Array.set(arr, index, 0);
```

Comment:
Avoid using Array#set - either use the if-else for Int instance, or extract to two separate methods